### ISSUE TITLE
Upload status seems broken

### ISSUE THREAD

**Issue by jbothma** [openspending/openspending#1354]
```
datapackage==1.1.4
datapackage-pipelines==1.5.3
datapackage-pipelines-aws==0.0.20
datapackage-pipelines-fiscal==0.1.11
```

When uploading a dataset using dpp the last step, reporting loading progress, seems to be stuck at 0

```
INFO    :Main                            :fiscal.upload: DEBUG   :***************************** [200] OK - GET: /package/status (begin) ******************************
INFO    :Main                            :fiscal.upload: DEBUG   :Request endpoint: https://openspending.org/package/status
INFO    :Main                            :fiscal.upload: DEBUG   :Request time: 0:00:00.239946
INFO    :Main                            :fiscal.upload: DEBUG   :Request parameters: {'datapackage': 'https://s3.amazonaws.com/datastore.openspending.org/b9d2af843f3a7ca223eea07fb608e62a/estimates-of-provincial-expenditure-south-africa-2018-19/datapackage.json'}
INFO    :Main                            :fiscal.upload: DEBUG   :Request payload: None
INFO    :Main                            :fiscal.upload: DEBUG   :Request headers: None
INFO    :Main                            :fiscal.upload: DEBUG   :Response headers: {'Date': 'Wed, 02 May 2018 16:23:33 GMT', 'Content-Type': 'application/json', 'Content-Length': '47', 'Connection': 'keep-alive', 'Set-Cookie': 'session=edb6f1f8-2fd1-48e6-8788-beeeffe9b090; Expires=Sat, 02-Jun-2018 16:23:32 GMT; HttpOnly; Path=/', 'Expect-CT': 'max-age=604800, report-uri="https://report-uri.cloudflare.com/cdn-cgi/beacon/expect-ct"', 'Server': 'cloudflare', 'CF-RAY': '414bd5420d26808e-CPT'}
INFO    :Main                            :fiscal.upload: DEBUG   :Response payload: {'progress': 0, 'status': 'loading-data'}
INFO    :Main                            :fiscal.upload: DEBUG   :Response cookies: <RequestsCookieJar[<Cookie session=edb6f1f8-2fd1-48e6-8788-beeeffe9b090 for openspending.org/>]>
INFO    :Main                            :fiscal.upload: DEBUG   :Request full URL: https://openspending.org/package/status?datapackage=https%3A%2F%2Fs3.amazonaws.com%2Fdatastore.openspending.org%2Fb9d2af843f3a7ca223eea07fb608e62a%2Festimates-of-provincial-expenditure-south-africa-2018-19%2Fdatapackage.json
INFO    :Main                            :fiscal.upload: DEBUG   :{
INFO    :Main                            :fiscal.upload: DEBUG   :    "progress": 0,
INFO    :Main                            :fiscal.upload: DEBUG   :    "status": "loading-data"
INFO    :Main                            :fiscal.upload: DEBUG   :}
INFO    :Main                            :fiscal.upload: DEBUG   :****************************** [200] OK - GET: /package/status (end) *******************************
INFO    :Main                            :fiscal.upload: DEBUG   :Saved request + response to /home/jdb/.gobble/GET.package.status.json
INFO    :Main                            :fiscal.upload: DEBUG   :estimates-of-provincial-expenditure-south-africa-2018-19 is loading (loading-data) 0/8
```
In my profile view it says something failed
```
Estimates of Provincial Expenditure of South Africa 2018-19 | Published | One resource / ~409374 records / 69.5 MB Updated at 2018/05/02 18:17 Edit Delete Download  More | ×FailedFailed to load data from https://openspending.org/package/status?datapackage=https%3A%2F%2Fdatastore.openspending.org%2Fb9d2af843f3a7ca223eea07fb608e62a%2Festimates_of_provincial_expenditure_of_south_africa_2018_19%2Ffinal%2Fdatapackage.json
-- | -- | -- | --
```
https://i.imgur.com/I4cyRJI.png

But it looks like my changes are actually online in the public view

**Comment by jbothma** [comment]
Nope, it really seems to be broken - the dataset isn't updated. I'm currently unable to fix a typo in the dataset which means I can't get data for a certain jurisdiction in my visualisations.

**Comment by brew** [comment]
Closed as a duplicate of #1367.
